### ISSUE TITLE
Set environment variable values for more than just Docker Compose

### ISSUE THREAD

**Issue by nickwesselman** [Sitecore/docker-tools#1]
_As a developer, I'd like to use the `Set-DockerComposeEnvFileVariable` function for other dotenv files, like those used with Next.js, so that I can initialize values for developer environments._

Currently `Set-DockerComposeEnvFileVariable` only allows files named `.env`, and has a name that indicates it is only valuable for Docker Compose. Would like to use this function to initialize `NODE_EXTRA_CA_CERTS` for Next.js as well, e.g.

```
    Set-EnvFileVariable -Variable "NODE_EXTRA_CA_CERTS" `
         -Value "$(& $mkcert -CAROOT)\rootCA.pem" `
         -Path "$PSScriptRoot\src\rendering\.env.local"
```

If the function is renamed, the existing name should be retained as well as a pass-through for backward compatibility.

**Comment by nickwesselman** [comment]
closed with #2

**Comment by nickwesselman** [comment]
Will close when merged to master
